### PR TITLE
Updates README

### PR DESCRIPTION
**Pull request by FabianCernota**
I'm not totally sure if this is now the correct way to build. Because the old way didn't work anymore.

### CODE REVIEW

**Comment by taktv6** [comment]
Although "go build" works fine here the intended Build system is bazel.
It's not just for testing. "go test" works as well.

File: README.md [hunk 1]
```diff
@@ -9,17 +9,26 @@ A re-implementation of BGP, IS-IS and OSPF in go. We value respect and robustnes
...
 Build
 -----
 
-    bazel build //:bio-rd
-    bazel-bin/linux_amd64_stripped/bio-rd -arguments go -here
-
-or
-
-    bazel run //:bio-rd -- -arguments go -here
+BGP
```
Comment (awlx) [inline README.md]: Simple bazel build //... builds all of it.

File: README.md [hunk 2]
```diff
@@ -13,22 +13,12 @@ We use [Bazel](https://bazel.io) to test bio-rd.
 
 Build
 -----
-
-BGP
----
+ 
 ```
-cd examples/bgp
 bazel build //...
```
Comment (awlx) [inline README.md]: You can run Bio-rd with
```
bazel-bin/<arch>_stripped/bio-rd
```

File: README.md [hunk 3]
```diff
@@ -13,22 +13,12 @@ We use [Bazel](https://bazel.io) to test bio-rd.
...
-./bgp
 ```
+Now you can find binaries under bazel-bin/examples/
```
Comment (awlx) [inline README.md]: Now you can find examples in bazel-bin/examples

File: README.md [hunk 4]
```diff
@@ -9,17 +9,16 @@ A re-implementation of BGP, IS-IS and OSPF in go. We value respect and robustnes
...
 ========
 
-We use [Bazel](https://bazel.io) to build bio-rd.
+We use [Bazel](https://bazel.io) to test bio-rd.
 
 Build
 -----
+ 
+```
+bazel build //...
```
Comment (awlx) [inline README.md]: You can run bio-rd with ```bazel-bin/<arch>_stripped/bio-rd -arguments go -here```

**Pull request closed without merging**